README.md

```diff
@@ -118,14 +85,29 @@ using (Image<Rgba32> image = new Image<Rgba32>(400, 400))
 }
 ```
 
-For optimized synchronous access within a loop it is recommended that the following methods are used.
+`Rgba32` is our default PixelFormat, equivalent to `System.Drawing Color`. For advanced pixel format usage there are multiple [PixelFormat implementations](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/PixelFormats) available allowing developers to implement their own color models in the same manner as Microsoft XNA Game Studio and MonoGame. 
 
-1. `image.GetRowSpan(y)`
-2. `image.GetRowSpan(x, y)`
+All in all this should allow image processing to be much more accessible to developers which has always been my goal from the start.
 
-For advanced pixel format usage there are multiple [PixelFormat implementations](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/PixelFormats) available allowing developers to implement their own color models in the same manner as Microsoft XNA Game Studio and MonoGame. 
+Check out [this blog post](https://sixlabors.com/blog/announcing-imagesharp-beta-1/) or our [samples repository](https://github.com/SixLabors/ImageSharp/tree/master/samples) for more examples!
```

Comment:
I've created a new repo for our samples you will want to use this instead.
https://github.com/SixLabors/Samples/tree/master/ImageSharp

Reply:
Nice! It wasn't yet here when I was editing the document 😄

Reply:
yeah I know.. that's why I quickly pushed up a version ;)